moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -73,6 +78,13 @@ export default function ComponentListItem({
           </IconButton>
         </ListItemSecondaryAction>
       </ListItem>
+      {!isComponentMapped && (
+        <ListItem dense className={classes.listItem}>
+          <ListItemText
+            primary={<Alert severity="error">Component is not mapped</Alert>}
```

Comment:
This works for me -- I was considering proposing a call to action of sorts, "add component to map", but that should only be to the owner of the project and I am hoping that its self evident that that is how to remove the alert.

Reply:
I like that call to action too if we are trying to motivate users to make the alert go away by mapping. Also wanted to see what y'all think about updating the icon next to the component type when not mapped, and then showing the alert when it is expanded? Not sure about the double icons though.

Something like:
<img width="343" alt="Screenshot 2023-10-09 at 10 37 29 AM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/290e19bf-657b-45ed-81f2-1e380875f016">

Then expanded:
<img width="344" alt="Screenshot 2023-10-09 at 10 40 40 AM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/36073165-c448-47d0-91e6-af5d8d6070b5">

Reply:
oooh i love it! will make that change 🚀

Reply:
@mddilley do you have this coded out? if so...push?

Reply:
git stash pop and pushed ✅